File: src/main/kotlin/com/itmo/microservices/demo/order/api/dto/OrderItem.java
package com.itmo.microservices.demo.order.api.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class OrderItem {
    private UUID uuid;
    private String title;
    private int price;
}

File: src/main/kotlin/com/itmo/microservices/demo/order/api/dto/OrderItem.kt
package com.itmo.microservices.demo.order.api.dto

import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Data
import lombok.NoArgsConstructor
import java.util.*

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
class OrderItem {
    private val uuid: UUID? = null
    private val title: String? = null
    private val price = 0
}